Flow improvement (#43)
githedgehog/docs: docs/vlab/overview.md

KVM acceleration can be used
```

> ### ORAS
> For convenience HedgeHog provides a script to install oras:
> ```bash
> curl -fsSL https://i.hhdev.io/oras | bash
> ```

Reviewer: ````suggestion
### ORAS

For convenience Hedgehog provides a script to install oras:

```bash
curl -fsSL https://i.hhdev.io/oras | bash
```
````

I don't think we spell `HedgeHog`.

Also regarding the case - Is it `ORAS` (heading) or `oras` (paragraph)?

Author: so, I think when talking about the project we spell it as ORAS which matches how the project itself spells it. When using it as a title, Oras, and when referring to the cli command, `oras` or however it would be typed into a shell to execute.

> ### HHFAB
> A github access token is required to download `hhfab`, please submit a ticket using the [Hedgehog Support Portal](https://support.githedgehog.com/). Once obtained, use the provided username and token to log into the Github container registry:
> ```bash
> docker login ghcr.io --username provided_username --password provided_token
> ```
> Hedgehog maintains a utility to install and configure VLAB, called `hhfab`. To install:
> ```bash
> curl -fsSL https://i.hhdev.io/hhfab | bash
> ```

Reviewer: ````suggestion
### HHFAB

You need a GitHub access token to download `hhfab`, please submit a ticket using the [Hedgehog Support Portal](https://support.githedgehog.com/). Once obtained, use the provided username and token to log into the GitHub container registry:

```console
$ docker login ghcr.io --username provided_username --password provided_token
```

Hedgehog maintains a utility to install and configure VLAB, called `hhfab`. To install:

```console
$ curl -fsSL https://i.hhdev.io/hhfab | bash
```
````

Also: HHFAB vs. hhfab

Author: see oras discussion above.

KVM acceleration can be used
```

### Oras

For convenience Hedgehog provides a script to install `oras`:

```bash
curl -fsSL https://i.hhdev.io/oras | bash
```

### Hhfab

> Hedgehog maintains a utility to install and configure VLAB, called `hhfab`. To install:

Author: No pain here. I appreciate the specificity.

```bash
docker login ghcr.io --username provided_username --password provided_token
```

> Hedgehog maintains a utility to install and configure VLAB, called `hhfab`. To install:

Reviewer: > Hedgehog maintains a utility to install and configure VLAB, called `hhfab`.

I'd move that to the top of the subsection, before mentioning the token required to download this `hhfab` you mention :)

Also s/github/GitHub/

KVM acceleration can be used
```

### Oras

For convenience Hedgehog provides a script to install `oras`:

```bash
curl -fsSL https://i.hhdev.io/oras | bash
```

### Hhfab

Hedgehog maintains a utility to install and configure VLAB, called `hhfab`.

> A GitHub access token is required to download `hhfab`, please submit a ticket using the [Hedgehog Support Portal](https://support.githedgehog.com/). Once obtained, use the provided username and token to log into the GitHub container registry:
> 
> To install:
> ```bash
> docker login ghcr.io --username provided_username --password provided_token
> ```
> 
> ```bash
> curl -fsSL https://i.hhdev.io/hhfab | bash
> ```

Reviewer: ````suggestion
You need a GitHub access token to download `hhfab`, please submit a ticket using the [Hedgehog Support Portal](https://support.githedgehog.com/). Once in possession of the credentials, use the provided username and token to log into the GitHub container registry:

```bash
docker login ghcr.io --username provided_username --password provided_token
```

Once logged in, download and run the script:

```bash
curl -fsSL https://i.hhdev.io/hhfab | bash
```
````

KVM acceleration can be used
```

### Oras

For convenience Hedgehog provides a script to install `oras`:

```bash
curl -fsSL https://i.hhdev.io/oras | bash
```

### Hhfab

Hedgehog maintains a utility to install and configure VLAB, called `hhfab`.

You need a GitHub access token to download `hhfab`, please submit a ticket using the [Hedgehog Support Portal](https://support.githedgehog.com/). Once in possession of the credentials, use the provided username and token to log into the GitHub container registry:

```bash
docker login ghcr.io --username provided_username --password provided_token
```

Once logged in, download and run the script:

```bash

> curl -fsSL https://i.hhdev.io/hhfab | bash

Reviewer: Side note, we may have to change this domain in the future (https://every.to/p/the-disappearance-of-an-internet-domain).

Reviewer 2: looks like we're good for a few years with it, but that's a danger of ccTLDs :(
